PowerShellEditorServices.build.ps1

```diff
@@ -19,19 +19,19 @@ param(
 #Requires -Modules @{ModuleName="InvokeBuild";ModuleVersion="3.2.1"}
 
 $script:IsUnix = $PSVersionTable.PSEdition -and $PSVersionTable.PSEdition -eq "Core" -and !$IsWindows
```

Comment:
Honestly this test is silly, I'd change it to:

```powershell
$script:IsUnix = $IsLinux -or $IsMacOS
```

Or:

```powershell
$script:IsUnix = $IsWindows -eq $false
```

Reply:
Agreed. Also `Unix` is wrong, it's just `*nix` so I changed to `IsNix`. Neither of these platforms are Unix.